addons/website/models/website.py

```diff
@@ -919,6 +919,9 @@ def get_website_meta(self):
             'twitter_meta': twitter_meta
         }
 
+    def _get_relative_url(self, url):
```

Comment:
Need to move up in `Website` class, this is inside `SeoMetadata` class

Reply:
thanks ;) 
A diff of 2 line s to backport from v13 to v12, and 2 conflicts ;)

Fixed